### PR TITLE
change memcached cache chart to stacked type

### PR DESCRIPTION
**Pull request by paulfantom**
Apply @ktsaou suggestion from #831.

### CODE REVIEW

File: python.d/memcached.chart.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 CHARTS = {
     'cache': {
-        'options': [None, 'Cache Size', 'kilobytes', 'Cache', 'memcached.cache', 'stacked'],
+        'options': [None, 'Cache Size', 'megabytes', 'Cache', 'memcached.cache', 'stacked'],
         'lines': [
             ['bytes', 'used', 'absolute', 1, 1048576],
```
Comment (ktsaou) [inline python.d/memcached.chart.py]: `bytes` or `used`

File: python.d/memcached.chart.py [hunk 2]
```diff
@@ -24,7 +24,7 @@
...
             ['available', 'available', 'absolute', 1, 1048576]
```
Comment (ktsaou) [inline python.d/memcached.chart.py]: Again for compatibility, please make id = `avail` and name = `available`.

**Comment by paulfantom** [comment]
Sorry, I didn't refresh page before merge. Changing it now.

File: python.d/memcached.chart.py (hunk 1 above)
Comment (paulfantom) [inline python.d/memcached.chart.py]: Usually I take id from data returned by application (memcached) and assign custom name. In this case memcached returns used bytes as `bytes`, but for user it is better to see it as `used`. But I changed it in [last commit to master branch](https://github.com/firehol/netdata/commit/7442a84d723eb8eada6ba58359d6e808a4c1e4df).

**Pull request merged**